### ISSUE TITLE
Plans to streamline future platform maintenance.

### ISSUE THREAD

**Issue by Zimmi48** [coq/platform#152]
Following a discussion with @palmskog today, we considered the following steps that could help streamline future platform maintenance:

- We should document who the platform maintainers are. A proposal is to add (somewhere in the README) something resembling the following:

  Platform maintainers:
  - Michael Soegtrop (leading the platform effort)
  - Enrico Tassi
  - Théo Zimmermann

  Liaison with the opam-coq-archive:
  - Karl Palmskog

- We should document the process in the platform repo. Something that would expand on the platform charter, CEP 52 and current technical documentation. IMHO we should split this documentation based on the target audience:

  - Platform maintainers
  - Maintainers of Coq packages (included or to be included in the platform)
  - People who want to ship the platform in their distribution
  - Platform users

- We should determine where everyone can help and how integration decisions are taken (type of PR that could be merged by @gares for instance, type of PR that needs to be approved by @MSoegtropIMC).

Given the centrality of the platform in the new release model, we shouldn't put all the burden of maintaining the platform on @MSoegtropIMC alone. OTOH, not having a sufficiently detailed documented process creates a barrier for others to get involved (to avoid doing something differently from what was planned).

Having a checklist of tasks that need to be done for the next platform release (like we have for Coq releases) could be helpful to follow the process.

**Comment by MSoegtropIMC** [comment]
Yes a documentation update would help and I would appreciate some support. Although I guess things get easier after all the infrastructure updates are done. I am currently reworking the documentation btw. Let's have a discussion after I pushed this (expected early next week).

**Comment by MSoegtropIMC** [comment]
Triage note: assigned to @rtetley
